package.json

```
@@ -4,14 +4,18 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "assemble": "./assemble.js",
+    "assemble": "npm-run-all --serial assemble-local format",
+    "assemble-local": "npm-run-all --parallel -r worker assemble-cmd",
+    "assemble-cloud": "npm-run-all --parallel -r worker assemble-cloud-cmd",
+    "assemble-cmd": "./assemble.js",
+    "assemble-cloud-cmd": "./assemble.js --cloud",
     "build": "docusaurus build",
     "clear": "docusaurus clear",
     "deploy": "docusaurus deploy",
     "docusaurus": "docusaurus",
     "format": "dprint fmt",
-    "gen": "snipsync && ./assemble.js && dprint fmt && docusaurus build",
-    "gen-cloud": "snipsync && ./assemble.js --cloud && dprint fmt && docusaurus build",
+    "gen": "npm-run-all --serial snipsync assemble-local format build",
+    "gen-cloud": "npm-run-all --serial snipsync assemble_cloud format build",
```

```suggestion
    "gen-cloud": "npm-run-all --serial snipsync assemble-cloud format build",
```

Should use `-` so that it calls `assemble-cloud`